source/common/http/conn_manager_impl.cc

```diff
@@ -437,7 +437,7 @@ void ConnectionManagerImpl::ActiveStream::decodeHeaders(HeaderMapPtr&& headers,
   // here and keep it under 60K. Ultimately it would be nice to bring this to a lower value but
   // unclear if that is possible or not.
   if (request_headers_->byteSize() > (60 * 1024)) {
-    HeaderMapImpl headers{{Headers::get().Status, std::to_string(enumToInt(Code::BadRequest))}};
+    HeaderMapImpl headers{{Headers::get().Status, std::to_string(enumToInt(Code::RequestHeaderFieldsTooLarge))}};
```

Comment:
@alyssawilk FYI, the comment above about nghttp2 is out of date. They added a fix based on an issue I opened a long time ago: https://nghttp2.org/documentation/nghttp2_option_set_max_send_header_block_length.html

If we wanted to change any of the sizes, we can now pretty easily test stuff. Not really suggesting any change in this PR, other than maybe a comment update if you feel like it.